### ISSUE TITLE
[PATCH] LaTeX with CREATE_SUBDIRS=YES (Origin: bugzilla #756737)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#5889]
status NEEDINFO severity *normal* in component *documentation* for *---*
Reported in version *1.8.10* on platform *Other*
Assigned to: Dimitri van Heesch

Original attachment names and IDs:
- _[LaTeX_with_CREATE_SUBDIRS=YES.patch](https://sourceforge.net/projects/doxygen/files/attachments/bug756737_313512_LaTeX_with_CREATE_SUBDIRS=YES.patch/download/download)_ (ID 313512)

On 2015-10-17 06:41:28 +0000, k.ohgishi@gmail.com wrote:

    Created attachment 313512
    remove stripPath and changed from hyperlink to hyperref
    
    The output includes labels with both strip path and don't strip path when
    CREATE_SUBDIRS=YES. So fix to remove stripPath.
    
    The output with hyperlink doesn't jump. So output are changed to hyperref.

On 2015-12-28 20:53:14 +0000, Dimitri van Heesch wrote:

    If I test your changes on a very simple example:
    
    --------------------------
    /** A class */
    class A
    {
    };
    
    /** Another class.
     *  @see A
     */
    class B
    {
    };
    --------------------------
    
    Then the reference to A in class B, gives:
    
    LaTeX Warning: Hyper reference `class_a' on page 3 undefined on input line
    7.
    
    So stripPath seems to be needed. Maybe you found a case where it is missing?

**Comment by albert-github** [comment]
I think this patch is obsolete.
- seen the problems as reported by @doxygen with the patch
- we now use everywhere in the latexcode for the links `hyperlink`
- all links work and jump to the right place (to the best of my knowledge)

**Comment by k-ohgishi** [comment]
I checked this issue was fixed at 1.8.12.

```
::::::::::::::
Makefile
::::::::::::::
DOXYGEN_DIR	= ~/doxygen-1.8.12/bin
#DOXYGEN_DIR	= ~/doxygen-1.8.11/bin
#DOXYGEN_DIR	= ~/doxygen-1.8.10/bin

all:
	LANG=C ${DOXYGEN_DIR}/doxygen
	grep --recursive --color test_8h latex

clean:
	$(RM) --recursive latex
::::::::::::::
Doxyfile
::::::::::::::
CREATE_SUBDIRS         = YES
QUIET                  = YES
INPUT                  = dir
GENERATE_HTML          = NO
HAVE_DOT               = NO
::::::::::::::
test.c
::::::::::::::
/**
 * @file
 */
/**
 * @brief main
 * @ref test
 */
int main(int argc, char *argv[])
{
	test();
	retrun 0;
}
::::::::::::::
dir/test.h
::::::::::::::
/**
 * @file
 **/

/**
 * @brief TEST
 **/
void test(void);
```
